docs/ui/editor.rst

```diff
@@ -0,0 +1,46 @@
+.. _ref_ui_editor:
+
+======
+Editor
+======
+
+.. image:: images/editor.png
+    :alt: The editor page in the EdgeDB UI, inside which the query builder
+          is shown as the user puts together a query to see the name property
+          for a user-defined type called Book. A filter on the object's id
+          and a limit to the number of object types returned are being set.
+          The Editor icon is a blue square resembling a pad, with an orange
+          line resembling a pencil on top.
+    :width: 100%
+
+Query Editor
+------------
+
+.. image:: images/editor.png
+    :alt: The Editor tab in the EdgeDB UI editor page, showing a query
+          appended with the analyze keyword to analyze performance. The
+          performance results show up in a graph on the right, with separate
+          colored rectangles for each link traversed by the query.
+    :width: 100%
+
+The query editor outwardly resembles the REPL with some auto-completion
+functionality, but is most useful when paired with the ``analyze`` keyword.
+Prepending ``analyze`` to the front of any query will display a visual
+analyzer to help you tweak the performance of your EdgeQL queries.
```

Comment:
```suggestion
analyzer to help you understand the performance of your EdgeQL queries.
```

—? We don't provide much active guidance in the analyzer, so I wouldn't want people to think `analyze` will actually help them tweak the performance directly. wdyt?

Reply:
I did steal the wording from [here](https://www.edgedb.com/docs/changelog/3_x) but you're right that it's always better to underpromise and overdeliver and "tweak" does sort of imply a fine-grained adjustment that `analyze` doesn't provide.